spec/controllers/single_use_links_viewer_controller_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
 
+# TODO: Useless test?
```

Comment:
This test is kinda pointless. Its basically checking if ruby assignment operator (=) works.
Controller Code:
```ruby
self.presenter_class = Hydranorth::GenericFilePresenter
```
Test Code:
```ruby
 expect(SingleUseLinksViewerController.presenter_class).to be Hydranorth::GenericFilePresenter 
```